cypress/e2e/1-getting-started/classTest.cy.js

```diff
@@ -1,11 +1,17 @@
-import  ContactPage  from '../../support/contacts/contact'
+import { ContactPg } from '../../support/contacts/contact'
```

Comment:
Brackets added. Their presence or absence should not affect the import, but I have added them for general practice.